docs/aurora/data-science/frameworks/pytorch.md

```diff
@@ -38,77 +36,194 @@ print(f'Device properties = {torch.xpu.get_device_properties()}')
+        ```
+
+Each Aurora node has 6 GPUs (also called "Devices" or "cards") and each GPU is composed of two tiles (also called "Sub-device"). By default, each tile is mapped to one PyTorch device, giving a total of 12 devices per node in the above output. 
+
+!!! note "`import intel_extension_for_pytorch as ipex`"
+    
+    Along with importing the `torch` module, you need to import the `intel_extension_for_pytorch` module in order to detect Intel GPUs as `xpu` devices. 
+
+    !!! warning
+    
+        It is highly recommended to import `intel_extension_for_pytorch` right after `#!python import torch`, prior to importing other packages, (from [Intel's "Getting Started" doc](https://github.com/intel/intel-extension-for-pytorch/blob/main/docs/tutorials/getting_started.md)).
+
+
+??? info "Using GPU Devices as PyTorch devices"
+
+    By default, each tile is mapped to one PyTorch device, giving a total of 12 devices per node, as seen above. 
+    To map a PyTorch device to one particular GPU Device out of the 6 available on a compute node, these 
+    environmental variables should be set
+    
+    ```bash
+    export ZE_FLAT_DEVICE_HIERARCHY=COMPOSITE
+    export ZE_AFFINITY_MASK=0
+    
+    # or, equivalently, following the syntax `Device.Sub-device`
+    export ZE_AFFINITY_MASK=0.0,0.1
+    ```
+    In the example given above, an application is targeting the `Device:0` 
+    and `Sub-devices: 0, 1`, i.e. *the two tiles of the GPU:0*. This is 
+    particularly important in setting a performance benchmarking baseline.
+    Setting the above environmental variables after loading the frameworks modules,
+    you can check that each PyTorch device is now mapped to one GPU:
+    ```python linenums="1"
+    import torch
+    import intel_extension_for_pytorch as ipex
+    torch.xpu.device_count()
+    torch.xpu.get_device_properties()
+    ```
+    
+    ???+ example "Example output"
+    
+        ``` { .bash .no-copy }
+    	1
+    	_XpuDeviceProperties(name='Intel(R) Data Center GPU Max 1550', platform_name='Intel(R) Level-Zero', type='gpu', driver_version='1.3.30872', total_memory=131072MB, max_compute_units=896, gpu_eu_count=896, gpu_subslice_count=112, max_work_group_size=1024, max_num_sub_groups=64, sub_group_sizes=[16 32], has_fp16=1, has_fp64=1, has_atomic64=1)
+    	```
+    	
+    More information and details are available through the [Level Zero Specification Documentation - Affinity Mask](https://oneapi-src.github.io/level-zero-spec/level-zero/latest/core/PROG.html?highlight=affinity#affinity-mask)
+
+
+
+## Code changes to run PyTorch on Aurora GPUs
+
+[Intel extension for PyTorch (IPEX)](https://pytorch.org/tutorials/recipes/recipes/intel_extension_for_pytorch.html) is an [open-source project](https://github.com/intel/intel-extension-for-pytorch) that extends PyTorch with optimizations for extra performance boost on Intel hardware, including Intel GPUs. 
```

Comment:
```suggestion
[Intel Extension for PyTorch (IPEX)](https://pytorch.org/tutorials/recipes/recipes/intel_extension_for_pytorch.html) is an [open-source project](https://github.com/intel/intel-extension-for-pytorch) that extends PyTorch with optimizations for extra performance boost on Intel hardware, including Intel GPUs. 
```

might want to rephrase the last part--- maybe to:
```suggestion
[Intel Extension for PyTorch (IPEX)](https://pytorch.org/tutorials/recipes/recipes/intel_extension_for_pytorch.html) is an [open-source project](https://github.com/intel/intel-extension-for-pytorch) that extends PyTorch with optimizations for extra performance boost on Intel CPUs and enables the use of Intel GPUs.
```

there is still no `xpu` functionality in upstream PyTorch yet, right?